src/plugin.py

```diff
@@ -262,6 +273,51 @@ async def pass_login_credentials(self, step, credentials, cookies):
 
         return self.authentication_client.parse_battletag()
 
+    async def get_friends(self):
+        if not self.authentication_client.is_authenticated():
+            raise AuthenticationRequired()
+        if not self.bnet_client.authenticated:
+            raise BackendError("not authenticated with battle.net")
+
+        friends_list = await self.social_features.get_friends()
+
+        return [UserInfo(user_id=friend.id.low, user_name=friend.battle_tag, avatar_url='', profile_url='') for friend_id, friend in friends_list.items()]
```

Comment:
are you sure empty string should be passed, not `None`? https://galaxy-integrations-python-api.readthedocs.io/en/latest/galaxy.api.html#galaxy.api.types.UserInfo

Reply:
Fixed.